src/core/dropdownmenu/dropdownmenu.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="dropdown is-hoverable">
+  <div class="dropdown-trigger">
+    <button class="button">
+      <span>{{ menuName }}</span>
+      <span class="icon is-small">
+        <i class="fas fa-angle-down"></i>
```

Comment:
We have no font awesome, use remix icons. Also ensure that the icon flips up when the dropdown is opened.